Library/Formula/fpp.rb

```diff
@@ -0,0 +1,23 @@
+class Fpp < Formula
+  homepage "https://facebook.github.io/PathPicker/"
+  head "https://github.com/facebook/pathpicker/"
+  version "0.0.3"
```

Comment:
Unless the URL is versioned in a convoluted way, `brew` will be able to read the version automatically. `brew audit` can point this out to you once you actually have a versioned tarball to compare against :)